libs/assets/src/lib/asset-details-table.tsx

```diff
@@ -105,7 +106,16 @@ export const rows: Rows = [
         return;
       }
 
-      return <ContractAddressLink address={asset.source.contractAddress} />;
+      return (
+        <>
+          <ContractAddressLink address={asset.source.contractAddress} />{' '}
```

Comment:
**question (non-blocking)**: I had missed this component. Is `links/eth-explorer-link` a complete duplicate? If so I'll ticket replacing that all over.

Reply:
Looks like it 👍🏻